src/index.js

```diff
@@ -11,14 +11,17 @@ var addDollarSign = function(value) {
 }
 
 exports.toCents = function(dollarString) {
-  if(typeof(dollarString) !== 'string') {
-    return dollarString;
+  if (!exports.valid(dollarString)) {
+    return 0;
```

Comment:
This is a public library and in a non-betting context (and perhaps even in a betting context) it is possible that 0 is a valid return value, in which case it would seem to be indistinguishable from the error condition. How about -1 for error?